modules/setup-openshift-oidc-provider.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-short-lived-tokens.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="setup-openshift-oidc-provider_{context}"]
+= Setup OpenShfit OIDC provider
+
+.Prerequisites
+* Administrative access to an OpenShift cluster.
+
+.Procedure
+. Follow the Google Cloud Platform documentation on how to link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Use workload identity federation for GKE].
```

Comment:
Is this procedure actually two steps? 

Step 1: Using workload identity federation for GKE
Step 2: Entering this command? 

Not sure how they're related. 

If it's actually just one step it will get a bullet point instead of a numeral: 

```suggestion
* Follow the Google Cloud Platform documentation on how to link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Use workload identity federation for GKE].
```

Reply:
This was still WIP, I removed the whole file for now.